test/System.IdentityModel.Tokens.Jwt.Tests/ExtensibilityTests.cs

```diff
@@ -133,6 +133,45 @@ public void NamedKeySecurityKeyIdentifierClause_Extensibility()
         }
 #endif
 
+        [Fact(DisplayName = "ExtensibilityTests: Algorithm names can be mapped inbound and outbound (AsymmetricSignatureProvider)")]
```

Comment:
FYI, these tests have been removed by @tushargupta51's latest PR. If you think it's not a good idea to re-introduce them, I'm fine.